### DISCUSSION TITLE
is there a https webserver-modul for micropython

### DISCUSSION THREAD

**Discussion by takkowa** [micropython/micropython#13017] [RP2040 / Pico]
Hi,

im looking for a webserver micropython-library/module that allows https especially for a raspberry pico.
I was searching for it in several forums and internapages, but I did not find any.
Does anybody know such a library/module?

**Comment by MrTinkerman** [comment]
I've been using Microdot.

I didn't try/use https but it seems to be supported:
[https://microdot.readthedocs.io/en/latest/extensions.html#https-support](https://microdot.readthedocs.io/en/latest/extensions.html#https-support)

**Reply by peterhinch** [reply]
Microdot is the one advocated by the maintainers.

**Comment by takkowa** [comment]
It took a little time to try the microdot server.
Generally it seems to do work as expected, as least I keept the example code running with ssl.
But on the other hand I failed with simple issues like reading more than one variable (user entered data) from a POST and store it into separate variables. 
Is there a more detailed description for the use of microdot, or any other Information (e.g. tutorial or book)  you would suggest.
